stakeholder-mapping/how-we-map.md

```diff
@@ -0,0 +1,26 @@
+## Process
+### Aim
+To have a comprehensive and diverse list of DT stakeholders (e.g. organisational, national, international) that include a) theme-specific stakeholders (e.g. NOC) and b) general stakeholders (DT Hub).
+
+### Method 
+A combination of desk research, targeted questions (e.g. to directors), and exploratory discussions with current stakeholders/partners (e.g. RAM meetings) were undertaken prior to September 2023. These activties identified a first set of stakeholders (n = 92) which were used to build the data collection structure. 
+
+The stakeholder list was developed based on the [RAM sharepoint people list]() and and informal list maintained by the RCM. The descriptors and organisartional units were developed as described above. 
+
+After this first round of mapping and dat collection, the TRIC-DT Hub team filled in missing values (e.g. email address or organisations) in the early list, and added new stakeholders based on their own desk or other research. The developed data collection structure was tested 
+
+#### Data Management
+The complied data will be GDPR Personal and may hold reputational risk. Yhe raw data will acconrdingly be held on Turing Sharepoint with restricted access. The team will determine what outputs of this exercise (in addition to documentatio) can be shared. 
+
+Chris B notes: "we should probably make sure that a DPAP has been conducted for the TRIC-DT (Janine or Karen should be able to support with this)." - UPDATE: see [issue #44](https://github.com/alan-turing-institute/tric-dt/issues/44)
```

Comment:
This looks like an internal comment, which we might want to cut or rethink in the public-facing repo?